server/application/application.proto

```diff
@@ -18,6 +18,7 @@ import "github.com/argoproj/argo-cd/reposerver/repository/repository.proto";
 message ApplicationQuery {
 	optional string name = 1;
 	required bool refresh = 2 [(gogoproto.nullable) = false];
+	repeated string project = 3;
```

Comment:
If this is a repeated string, i think name should be `projects`

Reply:
as we discussed offline used `[(gogoproto.customname) = "Projects"];`. This way generated field is going to be `Projects` and query url parameter remains 'project' (/applications?project=pr1&project=pr2)